soundbay/trainers.py

```diff
@@ -80,30 +81,39 @@ def train(self):
             if self.debug and epoch > 2:
                 break
 
+
     def train_epoch(self, epoch):
         self.model.train()
         for it, batch in tqdm(enumerate(self.train_dataloader), desc='train'):
-            if it == 3 and self.debug:
+            if it == 2:
                 break
 
             self.model.zero_grad()
-            audio, label = batch
-            audio, label = audio.to(self.device), label.to(self.device)
+            audio, label, raw_wav, idx = batch
+            audio, label, raw_wav, idx  = audio.to(self.device), label.to(self.device), raw_wav.to(self.device), idx.to(self.device)
 
+            if it == 1:
+                self.logger.upload_artifacts(audio, label, raw_wav, idx, sample_rate=self.train_dataloader.dataset.sample_rate, flag='train', epoch=epoch)
```

Comment:
could you please verify if works in debug mode as well? I mean, that the code works but nothing is uploaded ofcourse